autockt/envs/ngspice_ledro_d_fc.py

```
@@ -0,0 +1,365 @@
+        # print("num total:"+str(self.num_os))
+
+        # applicable only when you have multiple goals, normalizes everything to some global_g
+        self.specs_ideal_norm = self.lookup(self.specs_ideal, self.global_g)
+
+        # initialize current parameters
+        self.cur_params_idx = np.array([2] * 17)
+        self.cur_params_idx = np.array(
+            # [2, 2, 2, 2, 2, 2] + [200, 200, 200, 200, 200, 200] + [10, 10, 10, 10, 10]
+            [2, 2, 2, 2, 2, 2]
+            + [200, 200, 200, 200, 200, 200]
+            + [10, 10, 10, 10, 10]
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)